### DISCUSSION TITLE
Home Assistant OS 13.0.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3504] [Announcements]
## Home Assistant Operating System

* Use EROFS for the root filesystem (#3456) @sairon
* Bump buildroot to 2024.02.4 (#3487) @sairon
* Enable Netfilter MARK match kernel configuration (#3492) @agners
* Reduce log spam from Docker health checks (#3491) @sairon
* Generate version information for RAUC when rauc.db is empty (#3436) @sairon
* Remove old code from rauc-hook and GRUB env initialization (#3457) @sairon

## Raspberry Pi

* Patch genimage to force first usable LBA @ 34 to fix RPi 5/Windows issues (#3497) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Enable aQuantia AQtion ethernet driver (#3467) @sairon
* Fix boot slot A marked as bad on the very first boot (#3435) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Use GRUB2 legacy loader only on some Intel Atom boards specifically (#3498) @sairon
* Revert "Revert GRUB2 patch causing boot failure on some old x86_64 BIOSes (#3324)" (#3470) @sairon
* Enable aQuantia AQtion ethernet driver (#3467) @sairon
* Fix boot slot A marked as bad on the very first boot (#3435) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.44 (#3500) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.44 (#3500) @sairon
* Enable i6300ESB watchdog driver for generic-aarch64 (#3468) @sairon
* Populate grubenv with RAUC data in clean aarch64 OS image (#3458) @sairon
* Fix rauc hook to restore machine-id in new grub environment (#3451) @claplace

## Build System

* Test landing page is reachable without internet connection (#3499) @sairon
* Patch genimage to force first usable LBA @ 34 to fix RPi 5/Windows issues (#3497) @sairon

## Documentation

* Refer to GRUB instead of Barebox in top-level readme (#3464) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.0.rc1'>Home Assistant OS 13.0.rc1</a>.</em>

**Comment by tjorim** [comment]
No issues on a HA Yellow. Anything specific that needs testing?

**Comment by Anto79-ops** [comment]
just updated (from 12.4) on NUC 12. No issues to report.

**Comment by codyc1515** [comment]
Seems okay on RPi5.

**Comment by bschatzow** [comment]
No issues RPI 4 with SSD
